content/docs/0.16.x/reference/cli/commands/keptn_add-resource.md

```diff
@@ -19,7 +19,7 @@ From a technical perspective, the file provided via the *--resource* flag is sto
 
 **The target location of the resource:**
 
-- *--project* - is mandatory. The resource will be added to the root folder in the master branch. 
+- *--project* - is mandatory. The resource will be added to the root folder in the master branch. Do not use this command alone but add --all-stages if you are using resource service in branch mode.
```

Comment:
I'm not sure I understand what that added sentence is saying.  Maybe it's just me or maybe it could be reworded?

Reply:
We can change that maybe later on, all documentation changes in here are auto-generated from the keptn CLI so I don't want to change anything here.